### PR TITLE
Fulp changelogs now have their own folder for preservation

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

What it says on the tin. Our changelogs are now sent to `fulp_data/html/changelogs`, to be conveniently preserved along with `fulp_modules` during TGUs.

Consequences: 
 - A new workflow (Compile Changelogs (FULP)) has been added and will need to be enabled by creating a repository secret named `ACTION_ENABLER_FULP`. It can contain anything, it just has to exist.
 - This introduces a new TG edit on `tools/pull_request_hooks/autoChangelog.js`  to change the target folder for changelogs. Theoretically this could have been done without a TG edit, but would require making copies of several files that would then need to be maintained anyway, defeating the point.
## Why It's Good For The Game

We will finally have a proper changelog record for our PRs, since before that we just overwrote it every time we TGU'd.
This currently has no in-game implementation, but the whats and hows of that can be examined in the future.
## Changelog
:cl:
add: Creates a new fulp_data folder, for collecting various types of data that we may not want mixed with TG's.
fix: Fulp Changelogs are now logged in fulp_data/html/changelogs, to be preserved after TGUs. Hooray!
/:cl:

### CODE REVIEW

**Comment by OneAsianTortoise** [comment]
Fuck. Does that mean we have to properly write changelog now

File: fulp_modules/tg_edits.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 - tools/build/build.js > Adds our folder to be read for changes when compiling.
 
+- tools/pull_request_hooks/autoChangelog.js > Changes changelog folder to fulp_data/html/changelogs, to preserve them across TGUs.
+
 ## All Fulp files not contained within /fulp_modules/
```
[the remark this answers is not in the document]
Comment (FernandoJ8) [reply fulp_modules/tg_edits.md]: Fixed

**Comment by FernandoJ8** [comment]
To clarify, I haven't abandoned my currently open PRs

**Comment by FernandoJ8** [comment]
:/

**Pull request closed without merging**